src/Sentry/Laravel/Features/CacheIntegration.php

```diff
@@ -60,7 +60,7 @@ public function onBoot(Dispatcher $events): void
         if ($this->isTracingFeatureEnabled('redis_commands', false)) {
             $events->listen(RedisEvents\CommandExecuted::class, [$this, 'handleRedisCommands']);
 
-            $this->container()->afterResolving(RedisManager::class, static function (RedisManager $redis): void {
+            $this->container()->afterResolving(RedisFactoryContract::class, static function (RedisFactoryContract $redis): void {
                 $redis->enableEvents();
```

Comment:
The Factory does not contains `enableEvents` so we can't make this change as is: https://github.com/laravel/framework/blob/7db7ea950d3df663391718730f4490a4e189234a/src/Illuminate/Contracts/Redis/Factory.php.